Sources/Twift+Lists.swift

```diff
@@ -15,6 +15,7 @@ extension Twift {
   /// - Returns: A response object containing an array of Tweets, included expansions, and meta data for pagination
   public func getListTweets(_ listId: List.ID,
                             fields: Set<Tweet.Field> = [],
+                            userFields: Set<User.Field> = [],
```

Comment:
I think I see what's happened here; we should make `fields` a `Set<List.Field>` instead of introducing a `userFields` property.

Reply:
Yeah, never made these changes myself 😂 messed up the update and had some local git issues.